### ISSUE TITLE
Caracter not understood by cerberus

### ISSUE THREAD

**Issue by glassalle59** [cerberustesting/cerberus-source#2111]
I try to automate some tests on a react native application (ios side).

I have some erreur regarding getting xpath.

Inspector : 

![image](https://user-images.githubusercontent.com/33724879/76802573-59654180-67ae-11ea-9d50-5ac9f4d59f68.png)

I need to have some xpath with this character : 

A xpath like **//*[@label='APPROBATIONS ']** work well on Katalon (other automated tool) but not on Cerberus

Indeed, Cerberus transform the  on ?

_Error Cerberus : Failed to click because could not find element 'xpath=//*[@label='APPROBATIONS ?']

Can you fix this problem please ?

Thanks

**Comment by Nouxx** [comment]
Hi @glassalle59,

As a workaround, can you use the `contains()` xpath function to avoif checking with this caracter?

Something like `and contains(@label,'APPROBATION')]`

**Comment by glassalle59** [comment]
Yes but It will be easier if the Caracter is recognize by Cerberus
